tests/connect.php

```diff
@@ -92,6 +97,8 @@ public function should_throw_exception_when_trying_to_reopen_closed_connection()
             $this->expectException(InvalidOperationException::class);
 
             yield $connection->connectAsync();
+
+            Loop::cancel($watcher);
```

Comment:
see https://github.com/prooph/event-store-client/blob/02223cf5a0b13a95efe0c37226b7abd5aac772ac/tests/connect.php#L57

This trick doesn't work?

Reply:
I tried but the tests are failing with this.